plugins/MauticCrmBundle/Integration/SalesforceIntegration.php

```diff
@@ -2118,14 +2123,15 @@ protected function processCompositeResponse($response, &$totalUpdated = 0, &$tot
 
                     if ($integrationEntityId) {
                         /** @var IntegrationEntity $integrationEntity */
-                        $integrationEntity = $this->em->getReference('MauticPluginBundle:IntegrationEntity', $integrationEntityId);
+                        $integrationEntity = $this->em->find('MauticPluginBundle:IntegrationEntity', $integrationEntityId);
+                        if ($integrationEntity) {
+                            $integrationEntity->setLastSyncDate($this->getLastSyncDate());
+                            if (isset($this->salesforceIdMapping[$contactId])) {
```

Comment:
Duplicated code for lines https://github.com/mautic/mautic/pull/5160/files#diff-9b15b6be04482d6f57b012a98d72e58aR2088